bin/describe/describe_tests.ml

```diff
@@ -0,0 +1,156 @@
+open Import
+open Dune_rules
+
+module Test_description = struct
```

Comment:
One more field I would add here is the test alias. For the tests stanza this will just be `runtest-nameoftest`, same for inline tests and for cram tests it should be `cramtest` for `cramtest.t`.